Integreat/Integreat.Shared/Data/Loader/Targets/DisclaimerDataLoader.cs

```diff
@@ -10,15 +10,27 @@ namespace Integreat.Shared.Data.Loader.Targets
     /// <inheritdoc />
     public class DisclaimerDataLoader : IDataLoader
     {
-        public const string FileNameConst = "disclaimerV1";
-        public string FileName => FileNameConst;
+        private const string _FileNameConst = "disclaimerV3";
+        private string _FileName;
+
         public DateTime LastUpdated
         {
             get => Preferences.LastPageUpdateTime<Disclaimer>(_lastLoadedLanguage, _lastLoadedLocation);
             // ReSharper disable once ValueParameterNotUsed
             set => Preferences.SetLastPageUpdateTime<Disclaimer>(_lastLoadedLanguage, _lastLoadedLocation, DateTime.Now);
+        }
+
+        public string FileName
```

Comment:
Why not as auto-property and what kind of naming convention is "private string _FileName;"